SPLASH/src/game/events/NetworkWelcomeEvent.h

```diff
@@ -1,27 +1,13 @@
 #pragma once
-
-
-#include "../../../../Sail/src/Sail/events/Event.h"
-#include "../states/LobbyState.h"
 #include <list>
+#include "Sail/events/Event.h"
+#include "../states/LobbyState.h"
 
-class NetworkWelcomeEvent : public Event {
-public:
-	NetworkWelcomeEvent(std::list<Player> players) 
-		: Event(Event::NETWORK_WELCOME) 
-		, playerList(players)
-	{
-
-	}
-	~NetworkWelcomeEvent() {}
-
-	inline std::list<Player> getListOfPlayers() { return playerList; }
-
-	static Type GetStaticType() {
-		return Event::NETWORK_WELCOME;
-	}
-
-private:
-	std::list<Player> playerList;
```

Comment:
m_playerList? No big deal, but just sayin'. ^^

Reply:
Moved to public , therefore no "m_".
Replaced with a const list as well.